### ISSUE TITLE
Incorrect time format when using timeStep

### ISSUE THREAD

**Issue by chipit24** [dubert/react-kronos#57]
I have my component set up like so:
```
<Kronos
  time={this.state.selectedTime}
  format={'h:mm a'}
  onChangeDateTime={this.handleTime.bind(this)}
  placeholder="Time" />
```
Which results in the format of the time in the dropdown to be shown correctly:
<img width="117" alt="screen shot 2017-04-19 at 2 23 02 pm" src="https://cloud.githubusercontent.com/assets/7561061/25195484/c48c9208-250b-11e7-9320-eefa8d9e787b.png">

However, when I add `timeStep`, the time in the dropdown is now formatted in 24-hour time:
<img width="116" alt="screen shot 2017-04-19 at 2 33 24 pm" src="https://cloud.githubusercontent.com/assets/7561061/25195971/2cc761d0-250d-11e7-9f75-f02e8a269425.png">

The issue seems to be with this: https://github.com/dubert/react-kronos/blob/master/src/calendar.js#L244. The README mentions:
```
hour: - string : default: h:mm a
```
But looking at the source, it seems that `minute` is used, and the default is `HH:mm`, and not `h:mm a`.

To get things working how I want, I had to set:
```
options={{
  format: {minute: 'h:mm a'}
}}
```

**Comment by schuchowsky** [comment]
Had the same issue here.

**Comment by felixakiragreen** [comment]
_Just released **1.6.0**!_

This issue should be addressed, let me know if it is not.
